excel-importer/src/test/java/uk/gov/hmcts/ccd/definition/store/excel/endpoint/exception/SpreadsheetValidationErrorMessageCreatorTest.java

```diff
@@ -735,7 +736,7 @@ public void eventEntityInvalidUserRole_createErrorMessageCalled_customMessageRet
     public void eventEntityInvalidUserRole_createErrorMessageCalled_customMessageReturned_whenCaseRoleMissing() {
```

Comment:
Rename test when validation error `EventEntityInvalidUserRoleValidationError` is renamed

Reply:
Fixed in 3951e2b